code/datums/mind.dm

```diff
@@ -307,6 +307,9 @@
 /datum/mind/proc/remove_traitor()
 	remove_antag_datum(/datum/antagonist/traitor)
 
+/datum/mind/proc/remove_malf()
```

Comment:
Same here, this isn't necessary--just call it directly.

Reply:
Removed